Replicator/tests/ReplicatorLoopbackTest.hh

```diff
@@ -135,8 +135,10 @@ public:
         CHECK(_statusReceived.error.code == _expectedError.code);
         if (_expectedError.code)
             CHECK(_statusReceived.error.domain == _expectedError.domain);
-        CHECK(asVector(_docPullErrors) == asVector(_expectedDocPullErrors));
-        CHECK(asVector(_docPushErrors) == asVector(_expectedDocPushErrors));
+        if (!(_ignoreLackOfDocErrors &&_docPullErrors.empty()))
+            CHECK(asVector(_docPullErrors) == asVector(_expectedDocPullErrors));
```

Comment:
This seems to me that there is now no reason to set the expected doc errors and ALSO set ignore errors since ignore errors will just not check at all.  It seems to me like it should check that the doc errors are either empty, or match the expected ones.

Reply:
Nevermind I misread this. I'm going to merge it since this test is a constant bother.